longevity_large_partition_test.py

```diff
@@ -6,7 +6,7 @@
 class LargePartitionLongevityTest(LongevityTest):
 
     def test_large_partition_longevity(self):
-        compaction_strategy = self.params.get('compaction_strategy', default=CompactionStrategy.SIZE_TIERED.value)
+        compaction_strategy = self.params.get('compaction_strategy')
```

Comment:
You removed it, but did not go thru all test files that are using it, to see if there was no compaction strategy defined.

Reply:
@dkropachev 
I am not sure that I understand correctly this comment
Default compaction_strategy is defined in the test_defaults.py as SIZE_TIERED.